luigi/s3.py

```diff
@@ -284,23 +283,82 @@ def get_as_string(self, s3_path):
     def copy(self, source_path, destination_path, **kwargs):
         """
         Copy an object from one S3 location to another.
-
         :param kwargs: Keyword arguments are passed to the boto function `copy_key`
         """
         (src_bucket, src_key) = self._path_to_bucket_and_key(source_path)
         (dst_bucket, dst_key) = self._path_to_bucket_and_key(destination_path)
 
         s3_bucket = self.s3.get_bucket(dst_bucket, validate=True)
+        source_bucket = self.s3.get_bucket(src_bucket, validate=True)
+
+        multipart_size = 67108864
```

Comment:
Wait. This magic constant is (1) Copied twice in the code and (2) completely undocumented. Both are issues needing to be fixed.

Maybe this commit https://github.com/spotify/luigi/commit/7c089a91ffe070478c5814762c1ce6cc77aa8c8a can be inspiring? :)